docs/faq/env_var.md

```diff
@@ -218,6 +222,11 @@ When USE_PROFILER is enabled in Makefile or CMake, the following environments ca
   - When the array size is bigger than or equal to  this threshold, NDArray::Copy(from, to) is implemented by OpenMP with the Recommended OMP Thread Count.
   - When the array size is less than this threshold, NDArray::Copy(from , to)) is implemented by memcpy in single thread.
 
+* MXNET_OPTIMIZER_AGGREGATION_SIZE
+  - Values: Int ```(default=4)```
+  - Maximum value is 60.
```

Comment:
Maximum is here because the kernel cannot accept more than 60 pairs at the same time. This could be lifted by calling the kernel multiple times, although that's not ideal because setting it to a value higher than 60 would not have any performance benefit and it could actually slightly hurt performance because the operator would need to wait for more gradients to be ready before starting.

Reply:
That is why I decided to leave that limitation here and document it with the env variable, but I can change that if you feel it is better to not have such limitations.

Reply:
no, it should be fine to have the above limitation